From 7bec2bb2707d8afa18ca633fa882e5561dfd98be Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Tue, 21 May 2019 14:00:38 +0200
Subject: [PATCH 277/345] MIPS: OCTEON: Always try to allocate 1024 MB of
 32-bit memory.

Allocation of SWIOTLB area can fail with less than this amount.

Signed-off-by: David Daney <david.daney@cavium.com>
---
 arch/mips/cavium-octeon/setup.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/mips/cavium-octeon/setup.c b/arch/mips/cavium-octeon/setup.c
index 5e5df5972222..74d978731dee 100644
--- a/arch/mips/cavium-octeon/setup.c
+++ b/arch/mips/cavium-octeon/setup.c
@@ -74,7 +74,6 @@ const bool octeon_should_swizzle_table[256] = {
 EXPORT_SYMBOL(octeon_should_swizzle_table);
 
 static unsigned long long max_memory = ULLONG_MAX;
-static const unsigned long MIN_MEM_32 = 256 << 20;
 static unsigned long long reserve_low_mem;
 
 /*
@@ -1161,8 +1160,8 @@ void __init plat_mem_setup(void)
 
 	if (system_limit > max_memory)
 		system_limit = max_memory;
-	/* Try to get 512MB of 32-bit memory */
-	mem_32_size = 512 * (1 << 20);
+	/* Try to get 1024MB of 32-bit memory */
+	mem_32_size = 1024 * (1 << 20);
 
 	cvmx_bootmem_lock();
 	limit_max = 0xffffffffull;
-- 
2.25.1

